### ISSUE TITLE
Code and transformations for Imagenet

### ISSUE THREAD

**Issue by ghost** [ArchipLab-LinfengZhang/pytorch-self-distillation-final#5]
Hi,
Can you share the training code and transformations used on the Imagaenet dataset? I am unable to reproduce the paper results on Imagenet but could do it for CIFAR10/100.
I tried standard normalisation mean=[0.485, 0.456, 0.406] and std=[0.229, 0.224, 0.225] for the dataset.

Thanks,
Sidharth.

**Comment by xiaochenzhao-svg** [comment]
I also want to ask the same question; could you provide the code for ablation studies on ImageNet to prove the effectiveness of the method?
